### ISSUE TITLE
DROID won't run command line in windows if there is a space in the containing folder name

### ISSUE THREAD

**Issue by tnafrancesca** [digital-preservation/droid#888]
**Version and Environment**
6.6.0RCSnapshot3 with JRE (think this is consistent across other versions too) running in Windows 10

**What I did**
I ran the following command- 
C:\Users\fmackenzie\Downloads\droid-binary-6.6.0-rc3-SNAPSHOT-bin-win32-with-jre (1)>droid -h

**Expected Results**
I would have expected droid to show the help, or to run any other command line commands

**Actual Results**
I got the following error:
'C:\Users\fmackenzie\Downloads\droid-binary-6.6.0-rc3-SNAPSHOT-bin-win32-with-jre' is not recognized as an internal or external command,
operable program or batch file.

**Other information**
However C:\Users\fmackenzie\Downloads\droid-binary-6.6.0-rc3-SNAPSHOT-bin-win32-with-jre (1)>droid.bat  does open droid
so will open the GUI in the command line if there is a whitespace
By removing the whitespace for example:
C:\Users\fmackenzie\Downloads\droid-binary-6.6.0-rc3-SNAPSHOT-bin-win32-with-jre_1>droid -h would show the help. Which is why I believe the white space is the issue.
Would changing the arguments in the .bat file resolve this?

**Comment by steve-daly** [comment]
This is a Windows limitation rather than anything in DROID.  You have to put the command line in double quotes.  If you drag the file from Explorer to the command prompt, if will do this for you.

**Comment by steve-daly** [comment]
@tnafrancesca as I mention above, I don't think this is a DROID issue but just about how you run applications from the command line in Windows.  Shall we close this issue?

**Comment by steve-daly** [comment]
Having looked at this with @tnafrancesca, there is actually an issue with the DROID deployment when the application folder has a space in it.  The DROID GUI will launch fine, but there's an omission in droid.bat when passing command line parameters through. 

This section...

```
:PARAM
REM Has command-line parameters -- run command-line version:
%DROID_HOME%/jre/bin/java %DROID_OPTIONS% -jar "%DROID_HOME%droid-command-line-6.5.2.jar" %*

GOTO end

:NOPARAM
REM No command-line parameters passed -- run GUI version:
start "" "%DROID_HOME%/jre/bin/javaw" %DROID_OPTIONS% -jar "%DROID_HOME%droid-ui-6.5.2.jar"
```

...is missing double quotes around part of the command in the PARAM section where they are present in NOPARAM.  It should look like this, which I've tried and fixes the issues that @tnafrancesca was having.   It should look like this...

```
:PARAM
REM Has command-line parameters -- run command-line version:
"%DROID_HOME%/jre/bin/java" %DROID_OPTIONS% -jar "%DROID_HOME%droid-command-line-6.6.0-SNAPSHOT.jar" %*

GOTO end

:NOPARAM
REM No command-line parameters passed -- run GUI version:
start "" "%DROID_HOME%/jre/bin/javaw" %DROID_OPTIONS% -jar "%DROID_HOME%droid-ui-6.6.0-SNAPSHOT.jar"
```

**Comment by sparkhi** [comment]
This is fixed in 6.6.0 release
